### PR TITLE
In this feature tr y to implement the project requirment

### PR DESCRIPTION
**Pull request by Dagic-zewdu**
This project's requirements are done

- Created a database named `vet_clinic`.
- Created a table named `animals` with the following columns:
  - `id`: integer
  - `name`: string
  - `date_of_birth`: date
  - `escape_attempts`: integer
  - `neutered`: boolean
  - `weight_kg`: decimal
- Copied the SQL I used in the previous point into the `schema.sql` file in the generated repository.
- Inserted the following data:
  - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
  - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
  - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
  - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Copied the SQL I used in the previous point into the `data.sql` file in the generated repository.
- Wrote queries for the following:
  - Find all animals whose name ends in "mon".
  - List the name of all animals born between 2016 and 2019.
  - List the name of all animals that are neutered and have less than 3 escape attempts.
  - List the date of birth of all animals named either "Agumon" or "Pikachu".
  - List name and escape attempts of animals that weigh more than 10.5kg
  - Find all animals that are neutered.
  - Find all animals not named Gabumon.
  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
- Copied the SQL i used in the previous point into the `queries.sql` file in the generated repository.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts) VALUES ('Agumon',"Feb 3rd, 2020",10.23,0);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts) VALUES ('Gabumon',"Nov 15th, 2018",8,2);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts) VALUES ('Pikachu',"Jan 7th, 2021,",15.04,1);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts) VALUES ('Pikachu',"Jan 7th, 2021,",15.04,1);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts) VALUES ('Devimon',"Jan 7th, 2021,",15.04,1);
```
Comment (er-contreras) [inline data.sql]: - [ ] Don't forget to add if they were neutered. 😉

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @Dagic-zewdu, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Agumon',"Feb 3rd, 2020",10.23,0,1);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Gabumon',"Nov 15th, 2018",8,2,1);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Pikachu',"Jan 7th, 2021,",15.04,0);
+INSERT INTO animals (name,date_of_birth,weight_kg,escape_attempts,neutered) VALUES ('Devimon',"Jan 7th, 2021,",15.04,1);
```
Comment (TedLivist) [inline data.sql]: - [ ] Good job adding the values as requested, however, you need to ensure that your `neutered` values fit the criteria for **`Bit`** data types. Bit data types are **strings** of 1's and 0's, like so: `'1'`, `'0'`.

  neutered-bit

- [ ] **[OPTIONAL]** The previous reviewer probably missed it due to oversight. Your value for `date_of_birth` do not meet the **`Date`** data type. It should be something like so: `03/02/2020`.

  wrong-date

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Dagic-zewdu,

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Dagi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**